runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java

```diff
@@ -161,7 +161,7 @@ public Dimension render(Graphics2D graphics)
 				int size = npcComposition.getSize();
 				LocalPoint lp = npc.getLocalLocation();
 
-				Polygon tilePoly = Perspective.getCanvasTileAreaPoly(client, lp, size);
+				Polygon tilePoly = Perspective.getCanvasTileAreaPoly(client, lp, size, 0);
```

Comment:
Surely an extra overload method in `Perspective.java`

`public static Polygon getCanvasTileAreaPoly(
		@Nonnull Client client,
		@Nonnull LocalPoint localLocation,
		int size,
		int zOffset)
	{
    getCanvasTileAreaPoly(client, localLocation, size, 0);
}
`

Is better than adding a zero on every usage?